core/pom.xml

```diff
@@ -165,6 +170,7 @@
             <filtering>false</filtering>
             <includes>
                <include>schema/*-${infinispan.core.schema.version}.xsd</include>
+               <include>schema/*.proto</include>
```

Comment:
Updated to have:

```
schema/
schema/infinispan-config-10.0.xsd
schema/proto/
schema/proto/generated/
schema/proto/generated/persistence.core.proto
```

Reply:
Why not `proto/persistence.core.proto`, without the `schema` and `generated` directories?

Reply:
I included it in `schema`, because it's a schema. I can add it as just `root` if you prefer, but I don't think it makes any real difference.

The generated dir is included to allow us to be able to easily discern between generated and manually specified schemas. This was more of an issue when we had the AtomicMap marshallers etc, but I figured it was best to adopt an approach from the start that allows this kind of flexibility. I suppose we could remove the dir and just utilise the .proto filename.

Reply:
Ok, let's put them in `schema`, although https://developers.google.com/protocol-buffers/docs/proto never says "schema" ;)

But I'm really not convinced about the potential for confusion with non-generated proto files, since the generated files are all in `target`.

Reply:
> But I'm really not convinced about the potential for confusion with non-generated proto files, since the generated files are all in `target`.

I was thinking more in terms of the jar's content

Reply:
So in the end I went with `proto/` like you suggested :smile: